.github/workflows/run-fullnode-sync.yaml

```diff
@@ -32,17 +32,39 @@ on:
         type: number
         required: false
         default: 300 # Run for at most 5 hours
+  workflow_dispatch:
+    inputs:
+      TEST_NAME:
+        description: "The unique name of the fullnode test."
+        type: string
+        required: true
+      BRANCH:
+        description: "The aptos-core branch (or ref) to switch to before running the fullnode."
+        type: string
+        required: true
+      NETWORK:
+        description: "The network to connect the fullnode to: devnet, testnet, or mainnet."
+        type: string
+        required: true
+      BOOTSTRAPPING_MODE:
+        description: "The state sync bootstrapping mode for the fullnode."
+        type: string
+        required: true
+      CONTINUOUS_SYNCING_MODE:
+        description: "The state sync continuous syncing mode for the fullnode."
+        type: string
+        required: true
 
 jobs:
   fullnode-sync:
     runs-on: medium-perf-docker-with-local-ssd
-    timeout-minutes: ${{ inputs.TIMEOUT_MINUTES }}
+    timeout-minutes: ${{ inputs.TIMEOUT_MINUTES || 300 }}
```

Comment:
How about turning this into an input on `workflow_dispatch` too ?

Reply:
There were some issues with the type conversion between string and number actually. Need to dig more into what GH wants in each scenario lol